test/regressions/index.js

```diff
@@ -29,6 +29,9 @@ importRegressionFixtures.keys().forEach((path) => {
 const blacklist = [
   'docs-joy-getting-started-templates/TemplateCollection.png',
   'docs-joy-core-features-automatic-adjustment/ListThemes.png',
+  'docs-joy-tools/PaletteThemeViewer.png',
+  'docs-joy-tools/ShadowThemeViewer.png',
+  'docs-joy-tools/TypographyThemeViewer.png',
```

Comment:
It would be great if we add a motive on why we blacklist them. Outside of the first one, the other seems valuable, not sure.

```suggestion
  'docs-joy-getting-started-templates/TemplateCollection.png', // No public components
  'docs-joy-core-features-automatic-adjustment/ListThemes.png',
  'docs-joy-tools/PaletteThemeViewer.png',
  'docs-joy-tools/ShadowThemeViewer.png',
  'docs-joy-tools/TypographyThemeViewer.png',
```

Reply:
I don't see a value in having visual regression for the theme tokens. I will add comments on them.